dashboard/client/src/pages/serve/ServeSystemDetails.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  Box,
+  createStyles,
+  makeStyles,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@material-ui/core";
+import { Pagination } from "@material-ui/lab";
+import React, { ReactElement } from "react";
+import { RiErrorWarningFill } from "react-icons/ri";
+import { CollapsibleSection } from "../../common/CollapsibleSection";
+import { MetadataSection } from "../../components/MetadataSection";
+import { HelpInfo } from "../../components/Tooltip";
+import { ServeApplicationsRsp, ServeHttpProxy } from "../../type/serve";
+import { ServeHttpProxyRow } from "./ServeSystemDetailRows";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    table: {},
+    helpInfo: {
+      marginLeft: theme.spacing(1),
+    },
+    errorIcon: {
+      color: theme.palette.error.main,
+      width: 20,
+      height: 20,
+    },
+  }),
+);
+
+export type ServeDetails = Pick<
+  ServeApplicationsRsp,
+  "http_options" | "proxy_location"
+>;
+
+type ServeSystemDetailsProps = {
+  serveDetails: ServeDetails;
+  httpProxies: ServeHttpProxy[];
+  page: { pageSize: number; pageNo: number };
+  setPage: (key: string, value: number) => void;
+};
+
+const columns: { label: string; helpInfo?: ReactElement; width?: string }[] = [
+  { label: "Name" },
+  { label: "Status" },
+  { label: "Actions" },
+  { label: "Node ID" },
+  { label: "Actor ID" },
+];
+
+export const ServeSystemDetails = ({
+  serveDetails,
+  httpProxies,
+  page,
+  setPage,
+}: ServeSystemDetailsProps) => {
+  const classes = useStyles();
+
+  const isUnhealthy = httpProxies.some(({ status }) => status === "UNHEALTHY");
+
+  return (
+    <CollapsibleSection
+      title="System"
```

Comment:
Should we call this `Network`? `System` seems very broad of a term for just the HTTP proxy (although I guess it is a core component of serve)

Reply:
We'll be adding ServeController next, so System fits better after both those system components.